src/auth-service/models/Favorite.js

```
@@ -228,6 +228,178 @@
+
+      let filter = {
+        firebase_user_id: firebase_user_id,
+      };
+
+      let unsynced_favorite_places = (
+        await FavoriteModel(tenant.toLowerCase()).list({ filter }, next)
+      ).data;
+
+      unsynced_favorite_places = unsynced_favorite_places.map((item) => {
+        delete item._id;
+        return item;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)